### DISCUSSION TITLE
Replace Trimmed Area

### DISCUSSION THREAD

**Discussion by XVicarious** [ImageMagick/ImageMagick#1960] [General] [answered]
I could run 
 `convert image.png -fuzz X% -trim image-trim.png` 
 to trim borders off of an image. How can I do this, but replace what was trimmed with white for example? Assuming I don't know exactly how much I'm trimming, which edges (if any), and having to do it to hundreds of images.

**Comment by fmw42** [comment, marked as answer]
`convert image.png -fuzz X% -trim -background white -flatten image-trim.png`

If you do not use +repage after the trim, ImageMagick keeps track of the trimming information. So just flattening it against white should put white around the outside where the image was trimmed.

**Comment by XVicarious** [comment]
Ahha! I can conceptualize how this works a whole let better now, thank you!
